src/core/draw_tools/readme.md

```diff
@@ -0,0 +1,44 @@
+## v1
+
+based on: nebula.gl (deck.gl v8)
+
+integrated via MapboxLayer
+
+    // deck.gl v8
+    import {MapboxLayer} from '@deck.gl/mapbox'
+    map.addLayer(new MapboxLayer ...
+
+old docs: [pre-v1 structure](https://www.figma.com/file/G8VQQ3mctz5gPkcZZvbzCl/Untitled?node-id=0%3A1),
+[updated version](https://www.figma.com/file/FcyFYb406D8zGFWxyK4zIk/Untitled?node-id=0%3A1)
+
+### Toolbar button flow
+
+- [useDrawTools: controlsArray.[mode].action: toggleDrawMode(drawModes[mode]),](./index.ts#L128)
+- [toolboxAtom.toggleDrawMode](./atoms/toolboxAtom.ts#L64)
```

Comment:
Reminder: if you decide to change the name in toolboxAtom, also rename in readme. Otherwise ignore this